### PR TITLE
Milestone: Responsive Web Design

### PR DESCRIPTION
**Pull request by kwambiee**
# To set up the Desktop Version
I did the following:
1. I added markup to the homepage.
2. I added media query to css
3. I styled the elements to fit the desktop version

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -450,3 +474,261 @@ textarea:invalid:required {
   margin: 0 auto;
   width: 4rem;
 }
+
+@media screen and (min-width: 768px) {
+  body {
```
Comment (BenMukebo) [inline style.css]: - [x] the desktop size should be from 992px and above, please change the media query to be 
`@media screen and (min-width: 992px)`

File: index.html [hunk 2]
```diff
@@ -71,271 +80,308 @@ <h3>LET’S CONNECT</h3>
...
+                                    <button type="button" class="button-type">css</button>
+                                </li>
+                                <li>
+                                    <button type="button" class="button-type disable">javascript</button>
+                                </li>
+                            </ul>
+                        </div>
+                        <div class="btn-img">
+                            <button class="Tonic-project hidebtn" type="button">See Project</button>
+                            <img class="action-button" src="./Action-temp3.png" alt="">
```
Comment (BenMukebo) [inline index.html]: - [x] Please remove this image, all technologies should be either buttons or `a tag, not images.
- [x] Please adjust your indentation, you should use 2spaces only
btn img

File: style.css [hunk 3]
```diff
@@ -450,3 +474,261 @@ textarea:invalid:required {
...
+    margin: 0;
+  }
+
```
Comment (BenMukebo) [inline style.css:485]: - [x] Please adjust your styling, see this image below compared to the original design.
- [x]  Your paragraph text should be center, kindly fix that
- [x]  the paragraph font-weight does not much the design (yours is bold), consider using the same as provided in the template, please
- [x] the width of your input fields are not well adjusted, please fix them up
- [x]  Please add a placeholder to your `input name and email field` as you did with the `textarea`, kindly remove the value you just set to them.
center

- [x] It should be like this ⏬  instead 
foorm

- [x] Please give some breathing to this section, kindly  move these arrows a bit to the left as shown below ; following the figma template, you should consider using same padding ang margin.
arr

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi *** @kwambiee , ***

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight 💪🏻

- No linter errors. ✔️ 
- Github flow was correctly used.  ✔️   
- Good commit messages ✔️ 

 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_


- [x] Kindly, Update your README file, you are already working on the desktop version but the readme does not contain any information, please use this [template](https://github.com/microverseinc/readme-template) You can copy past but remember to customize it to your project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kwambiee** [comment]
@BenMukebo Well noted

File: style.css [hunk 4]
```diff
@@ -450,3 +474,290 @@ textarea:invalid:required {
...
+  input {
+    padding-right: 20rem;
+    border-radius: 0;
+  }
+
+  textarea {
+    border-radius: 0;
+    padding-right: 20rem;
+  }
```
Comment (sja-thedude) [inline style.css]: - [x] Kindly make sure to follow the Figma design guidelines and make the input field of all to equal.
- [x] Kindly make sure that paragraph is of the correct width.
- Your design:
Screenshot 2022-03-23 at 9 14 31 PM

- Figma design:
Screenshot 2022-03-23 at 9 17 28 PM

File: style.css [hunk 5]
```diff
@@ -450,3 +474,290 @@ textarea:invalid:required {
...
+
+  .submit {
+    padding: 1rem 1rem;
+    font-size: 17px;
+    font-weight: 500;
+  }
+
+  .divide {
+    display: none;
+  }
```
Comment (sja-thedude) [inline style.css:769]: - [ ] Kindly also make sure to fix mobile version text fields too.
- Your design:
Screenshot 2022-03-23 at 9 25 25 PM
- Figma design:
Screenshot 2022-03-23 at 9 25 06 PM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @kwambiee, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Professional read me

 ## Required Changes
- [x] _Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 4 above)
Comment (kwambiee) [reply style.css]: @BenMukebo Please run this link in your browser to confirm I have the right changes. 
https://raw.githack.com/kwambiee/First-Portfolio/desktop-version/index.html

**Review by aimemalaika — APPROVED** [review]
Hi @kwambiee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**